docs/changes/0.15.0a1.rst

```diff
@@ -0,0 +1,70 @@
+Changelog for QCoDeS 0.15.0a1
+========================================
```

Comment:
To avoid a warning I think you need to shorten the underline to match the text lenght

Reply:
Done. Thanks!